code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -101,6 +101,24 @@
 			new /obj/item/clothing/suit/hooded/chaplain_hoodie(src)
 			new /obj/item/weapon/card/id/syndicate(src)
 			return
+		if("mallninja")
+			new /obj/item/weapon/katana/syndicate(src)
+			new /obj/item/weapon/implanter/adrenalin(src)
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/storage/belt/military/assault(src)
+			new /obj/item/clothing/shoes/sandal/mninja(src)
+			new /obj/item/clothing/head/fedora/mninja(src)
+			new /obj/item/clothing/suit/jacket/leather/overcoat/mninja(src)
+			new /obj/item/clothing/under/pants/black(src)
+			new /obj/item/clothing/gloves/fingerless(src)
+			new /obj/item/clothing/glasses/sunglasses(src)
+			new /obj/item/clothing/mask/bandana/skull(src)
+			return

```

Comment:
Requesting a agebt ID card and a emag for quick entries and disguises (aka no AI track)